socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,30 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
-            )
+            self.quit_check()
+            try:
+                a_store.save_raw_and_processed(
+                    raw_crash,
+                    dump,
+                    processed_crash,
+                    crash_id
+                )
+            except Exception:
+                store_repr = getattr(
```

Comment:
No, it's not `repr` yet. At the moment, this variable is a pointer to a class. It *becomes* repr when you later use the `%r`. I suggest you simply call this variable `store_class` and on line 609 (and line 610) change it to:
```python
self.logger.error(
    '%r failed (crash ID: %s)',
    store_class,
    crash_id,
    exc_info=True
)
```
```